### ISSUE TITLE
Feat: The windows App lacks a title bar improvement

### ISSUE THREAD

**Issue by Nishchit14** [firecamp-dev/firecamp#186]
The windows build lacks a title bar.  
Thus, you cannot move, resize, minimize, etc. the window.

_Originally posted by @Alia5 in https://github.com/firecamp-dev/firecamp/discussions/167#discussioncomment-8547378_

**Comment by Alia5** [comment]
I'd be happy to help out by opening a PR.    

However, I don't think I have enough of an overview yet.

The quick-fix would be to set the `frame` property to true when calling `new BrowserWindow(...)`  
Is this an acceptable fix for the time being, if restricted to Windows?  
Or is it an issue on other platforms as well?

Should there even be a native Window-Frame shown, or is there a custom title-bar implemented that is simply now shown?

**Comment by teckdev** [comment]
Same issue is happening on Mac with M processor.

**Comment by Nishchit14** [comment]
@Alia5 I agree with your thought, I'll fix it by this weekend. It'll be available in the next release.

**Comment by MypowerHD** [comment]
Hey @Nishchit14, still present.

**Comment by Roguem99** [comment]
Still present on Windows

**Comment by MostHated** [comment]
Can't click on or edit the http:// address bar of a request either, making the app quite unusable.

**Comment by Lexazan** [comment]
still there, can't move or resize, app is unusable in that state.

**Comment by phillipvillaver** [comment]
Just downloaded the desktop client today. Firecamp 3.2.4 Windows client still has this issue. Here's a (quite cumbersome) workaround for the following actions:

**Restore (un-maximize) the window**, since the desktop client opens maximized by default
1. Focus on the Firecamp desktop client
2. Press Windows key + Down key

Window is now un-maximized but eats the entire screen. Drag the top/left/right/bottom part of the desktop client to resize lol

**Move the desktop client around.**

_Prerequisites: Restored Firecamp desktop client_

1. Focus on the Firecamp desktop client
2. Press the following keys in order:

a. Alt+Space
b. M
c. Down key

3. Move your mouse (don't click)
4. Left click when you're happy with the window position
